draft-brunstrom-taps-impl.md

```diff
@@ -330,6 +343,35 @@ Protocol options are checked next in order. Whether or not a set of protocol, or
 
 Branching for derived endpoints is the final step, and may have multiple layers of derivation or resolution, such as DNS service resolution and DNS hostname resolution.
 
+
+## Ranking Branches
```

Comment:
Can we call this step "sorting" rather than "ranking"? This would be more in line with Happy Eyeballs (https://tools.ietf.org/html/rfc8305#section-4)

Reply:
I am fine with renaming it – needs a little more explanation then, but we can do that

Reply:
Fixed with commit #9d3bf58